src/TestUtils/GcloudWrapper/CloudFunction.php

```diff
@@ -20,6 +20,7 @@
 use Symfony\Component\Process\Process;
 use Symfony\Component\Process\PhpExecutableFinder;
 use Symfony\Component\Process\Exception\ProcessFailedException;
+use PHPUnit\Framework\ExpectationFailedException;
```

Comment:
Why was this added?  I don't see it referenced in this class.

Reply:
Removed.